generators/templates.go

```diff
@@ -87,14 +87,70 @@ package client
 
 import (
 	"bytes"
+	"crypto/tls"
 	"encoding/json"
 	"errors"
 	"io/ioutil"
+	"os"
+	"os/user"
 	"net/http"
 
 	log "github.com/Sirupsen/logrus"
 )
 
+var httpClient *http.Client
+
+func init() {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+
+	httpClient = &http.Client{Transport: tr}
+}
+
+type NetctlConfig struct {
+	Token string ` + "`" + `json:"token"` + "`" + `
+}
```

Comment:
This is a generic go-client for contiv. We should avoid explicit references to netctl here.